### PR TITLE
Fixes #16376: 6.0 release in release-data

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16376

### CODE REVIEW

File: release-data/rudder-6.0.cson [hunk 1]
```diff
@@ -3,7 +3,10 @@
...
 
 # Status of the release (alpha, beta, rc, final)
-release-status = "beta"
+release-status = "final"
+
+# 9th December 2019
+release-date = "2019-12-09"
 
 # Has this version had Long-Term Support (LTS)?
 lts = false
```
Comment (VinceMacBuche) [inline release-data/rudder-6.0.cson]: Maybe we should change that too!

Comment (amousset) [reply release-data/rudder-6.0.cson]: It should not be used anymore, but I'm not sure we can safely remove it. I fear setting it to true would display "6.0 LTS" in some places.

Comment (VinceMacBuche) [reply release-data/rudder-6.0.cson]: It was done for 5.0 so i don't know :)

Comment (amousset) [reply release-data/rudder-6.0.cson]: Ok!

**Comment by amousset** [comment]
PR updated with a new commit

**Pull request merged**